src/integration/pages/articlePage/mundo.test.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * @pathname /mundo/articles/ce42wzqr2mko
```

Comment:
It would be interesting to see how this would work with multiple paths - or would we just create multiple files with slightly different names in order to test that page e.g. mundo.tc2.test.js?

Also keen to understand how we could potentially hook this into the existing Cypress config, to allow us to run multiple paths per service - before the latest refactor, I did a quick spike on this: https://github.com/bbc/simorgh/pull/5875/ - in particular https://github.com/bbc/simorgh/pull/5875/files#diff-6c8355b673b1f3b78693a1700601bb32

Reply:
I'm not sure what would be achieved with throwing multiple paths of the same service at at at a test file? We'd be testing the same code multiple times. It also wouldn't work with this setup because e.g. page title, headline, summary values etc would differ between each path. I do see value in testing the tc2 case though and for that I'd just create another file like you mentioned `mundoTC2.test.js` and maybe leave a comment in the test file explaining the difference.

With regards to mimicking the Cypress setup, at the last discussion we mentioned something along these lines:

> For now we shouldn't be testing 40+ services but should at least test a few with different things like LTR/RTL, services with variants etc. Worries about config bugs in services, although possible will be low risk and will be spotted by other means of testing and will be fixed.

If some testers and engineers disagree with this approach then we need to have another discussion and decide on another approach.